style.css

```diff
@@ -0,0 +1,398 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+html {
+  height: 100%;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  background-color: #F8F8FF;
+}
+
+h1, h3, a, p, li, label, .footer {
+  font-family: Arial;
+}
+
+.main-container {
+  margin: 0 auto;
+  max-width: 1200px;
+  -webkit-box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  -moz-box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  width: 100%;
+  min-height:100%;
+  position:relative;
+}
+
+.head-section {
+  padding: 30px;
+}
+
+h1 {
+  font-size: 2em;
+}
+
+h3 {
+  font-size: 1.4em;
+  display: inline-block;
+}
+
+img {
+  width: 100%;
+}
+
+.container {
+  padding: 30px 30px 80px 30px;
+}
+
+p, ul {
+  font-size: 1.2em;
+  line-height: 1.2em;
+}
+
+/* Nav */
+
+nav {
+  margin: 10px 0;
+  display: inline-block;
+  float: right;
+}
+
+#nav {
+  list-style: none;
+  padding: 0;
+}
+
+#nav li {
+  display: inline;
+}
+
+#nav li:nth-child(2) {
+  padding: 0 30px;
+}
+
+#nav li a {
+  display: inline-block;
+}
+
+/* end of Nav */
+
+/* Link styling */
+
+a  {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+a:visited {
+  color: #7B68EE;
+}
+
+a.link-color {
+  color: #778899;
+}
+
+a.link-color.active {
+  color: #7B68EE;
+}
+
+.head-section h1 a:visited {
+  color: #778899;
+}
+
+/* end of Link styling */
+
+/* home section */
+
+.container.home p:first-child {
+  margin-bottom: 15px;
+}
+
+.container.home h3 {
+  margin-bottom: 15px;
+}
+
+ul {
+  padding-left: 20px;
+  margin-bottom:15px;
+}
+
+/* end of home */
+
+/* about section */
+
+.container.about {
+  margin-top: 50px;
+}
+
+.img-circular{
+  width: 200px;
+  height: 200px;
+  background-image: url('me.jpg');
+  background-size: cover;
+  display: block;
+  border-radius: 200px;
```

Comment:
I could be wrong, but I'm fairly sure all browsers implement `border-radius` consistently now. Good move to use browser-specific on the box-shadow, but I think this one might be unnecessary these days.